test/test_runners.rb

```diff
@@ -18,9 +18,9 @@ def test_new
       state=nil
       assert_nothing_raised() { state=runner.run(spec) }
       assert_equal(1, state["steps"].size)
-      assert_equal(4, queue.size)
+      assert_equal(6, queue.size)
       assert_equal("started", queue.pop.text)
-      2.times{queue.pop}
+      4.times{queue.pop}
```

Comment:
Two more need to be popped so that the final message in the queue is the `finished` message.